imports/api/cases.js

```diff
@@ -310,6 +310,8 @@ Meteor.methods({
       normalizedParams.op_sys = 'Unspecified' // NOTE: This might an actual value at a later evolution step of the app
       normalizedParams[caseServerFieldMapping.category] = normalizedParams[caseServerFieldMapping.category] || '---'
       normalizedParams[caseServerFieldMapping.subCategory] = normalizedParams[caseServerFieldMapping.subCategory] || '---'
+      normalizedParams[caseServerFieldMapping.priority] = 'P3'
+      normalizedParams[caseServerFieldMapping.severity] = 'Normal'
```

Comment:
@sophieyang0813 do we have to specify that?
@nbiton if I remember correctly if these are not specified during case creation, then the BZ API is clever enough to pick whichever value has been defined as default in the BZFE

Reply:
> @nbiton if I remember correctly if these are not specified during case creation, then the BZ API is clever enough to pick whichever value has been defined as default in the BZFE

Correct

Reply:
yes, they should be removed so BZ can set the default

Reply:
Oh understood, it is BZ that sets the default values, not me. I thought it was my responsibility given the conversation on default values in #388. Will fix!

Reply:
> Is this within or beyond my scope of responsibility for this issue?

@sophieang0813 this beyond the scope of responsibility for this issue

FYI changing priority and severity can be done in the dashboard (BZFE)